docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -40,20 +24,18 @@ networkers=(
 )
 ----
 +
-** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the `Networker` node and its IP address.
+** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the Networker node and its IP address.
+
+.Procedure
 
-. Deploy the other `OpenStackDataPlaneNodeSet` CR. For simplicity, you can
-reuse most of the `nodeTemplate` section from the `openstack` CR designated for
-`Compute` nodes, though strictly speaking, some of the variables may be
-unnecessary because of a more limited set of services running on `Networker`
-nodes. Make sure `edpm_enable_chassis_gw: true` is set to run `ovn-controller`
-in gateway mode.
+. Deploy the `OpenStackDataPlaneNodeSet` CR for your Networker nodes:
 +
-. If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
+[NOTE]
+You can reuse most of the `nodeTemplate` section from the `openstack` CR that is designated for your `Compute` nodes. However, some of the variables might be unnecessary because of a more limited set of services that are running on `Networker` nodes. Ensure that `edpm_enable_chassis_gw: true` is set to run `ovn-controller` in gateway mode.
```

Comment:
By `openstack` CR, do you mean the `OpenStackDataPlaneNodeSet`CR that the customer created in the “Adopting Compute services to the RHOSO data plane” procedure? Is the CR always called `openstack` , or can customers name it whatever they want?

Reply:
yes, it is the CR from “Adopting Compute services to the RHOSO data plane”. The text here is more like: get the data from the CR sample you copied before and modify ...